src/CheckoutSdk.Extensions/CheckoutOptions.cs

```diff
@@ -8,6 +10,14 @@ public sealed class CheckoutOptions
 
         public string PublicKey { get; set; }
 
+        public string ClientId { get; set; }
+
+        public string ClientSecret { get; set; }
+
+        public Uri AuthorizationUri { get; set; }
+
+        public FourOAuthScope[] Scopes { get; set; }
```

Comment:
I made it an array because this maps the `json` file where literally there will be an array there. Probably the best option could be to have an ISet to match the existing implementation. I will have a look to see if it works.

Reply:
Ok, thanks